docs/docs/fallback-handoff.mdx

```diff
@@ -231,80 +215,147 @@ class ActionDefaultFallback(Action):
 ```
 
 
-
 <img alt="Intent Mappings" src={useBaseUrl("/img/intent_mappings.png")} width="240" />
 
-## Out of scope intent
 
-It is good practice to also handle questions you know your users may ask, but for which you haven't necessarily implemented a user goal yet.
 
-You can define an `out_of_scope` intent to handle generic out of scope requests, like “I'm hungry” and have
-the bot respond with a default message like “Sorry, I can't handle that request”:
+### Two-Stage Fallback
 
-```md
-* out_of_scope
-  utter_out_of_scope
-```
+In some cases, you will want to attempt to disambiguate the user's message by asking
+clarifying questions. The Two-Stage-Fallback is made to handle low NLU confidence in multiple stages
+using the following sequence:
+
+- If an NLU prediction has a low confidence score, the user is asked to affirm
+  the classification of the intent.  (Default action:
+  `action_default_ask_affirmation`)
+
+    - If they affirm by sending a message with high NLU confidence (e.g. by pressing
+      a button), the story continues as if the intent was classified
+      with high confidence from the beginning.
+    - If they deny by sending a message with the intent `out_of_scope`, the user is
+      asked to rephrase their message.
```

Comment:
can we shorten these? not sure the whole out_of_scope intent etc needs to be mentioned